### ISSUE TITLE
Chrome/NVDA only recognizes non-hidden text element

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#29]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1178" alt="Screenshot of NVDA in test environment. Speech viewer for NVDA is focusing on one element and reads graphic 3 Text" src="https://user-images.githubusercontent.com/98043076/151073420-c444ba61-15bb-4e35-bc06-37bf5ce584e6.png">

### Issue Summary
NVDA only reads the child element of what should be a paired second grouping. The user is not told this graphic element is part of a grouping. This issue is connected to the previous #28 issue (part of this grouping's pair is read in the "region" element text.)

### Video of issue
Example starts at 00:00:25
https://user-images.githubusercontent.com/98043076/151067013-96e52af1-f372-49a4-9ed3-a807136faac6.mp4

### Steps to reproduce
Use NVDA right arrow key to navigate into the SVG space. Press the arrow once more. There are only two steps (we would expect at least seven separate elements to navigate.)

### Expected behavior
- We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).

### Technical Details
Chrome Version 97.0.4692.99
NVDA 2021.3.1
Windows 11 Build 22000.438
